src/index.js

```diff
@@ -8,21 +8,26 @@ class VersionHash {
         this.options = Object.assign(
             {
                 length: 6,
-                delimiter: '.'
+                delimaiter: '.'
             },
             options
         )
 
         let delimiter = this.getDelimiter()
         const mixManifest = `${Config.publicPath}/mix-manifest.json`
-        const removeHashFromKeyRegex = new RegExp(delimiter + '(.+)\\.(.+)$', 'g')
+        const removeHashFromKeyRegex = new RegExp('(.*)\\' + delimiter + '(.*)\\.(.*)$', 'g')
```

Comment:
am not really sure what this regex does vs the current

Reply:
Well, the current one is simply not working ;-)

It's not even working when i pull the library. I always end up with `js` or `.css` and so on.

https://jsfiddle.net/tlapi/vox5y142/

Reply:
can u show me ur `webpack.mix.js` setup

Reply:
Sure. This means that I am getting different keys than you do, I suppose.

```
let mix = require('laravel-mix');
require('laravel-mix-versionhash');
const CleanWebpackPlugin = require('clean-webpack-plugin');

/*
 |--------------------------------------------------------------------------
 | Mix Asset Management
 |--------------------------------------------------------------------------
 |
 | Mix provides a clean, fluent API for defining some Webpack build steps
 | for your Laravel application. By default, we are compiling the Sass
 | file for the application as well as bundling up all the JS files.
 |
 */

mix.js('resources/assets/js/app.js', 'public/assets/js')
	.sourceMaps();
	
mix.versionHash();

mix.webpackConfig({
  plugins: [
    new CleanWebpackPlugin(['./public/assets/js'])
  ]
});
```

I gues this one is pretty standart.

Reply:
`laravel-mix@2.1.11`

Reply:
okay, try to use without the `sourceMaps` see if u r getting a diff results